english/tintumon-mallu/tintu-main.md

```diff
@@ -0,0 +1,2 @@
+In Mallu land there was a boy called [Tintu](../../Malayalam/malayalam.md)
+He was friends with [Maina](maina.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.